modules/react-native-status/ios/RCTStatus/RCTStatus.m

```diff
@@ -882,7 +882,7 @@ - (void) migrateKeystore:(NSString *)accountData
     return rootUrl.path;
 }
 
-RCT_EXPORT_BLOCKING_SYNCHRONOUS_METHOD(logFilePath) {
+RCT_EXPORT_BLOCKING_SYNCHRONOUS_METHOD(logFileDirectory) {
```

Comment:
I suppose this method should just return an empty string for it to work properly, checking now

Reply:
downstream (on `status-go` side) it is used as a relative path and it would result in some weird `/User/...the entire path from the root` path which will start in app's root dir (not system root). So if we just use `@""` here it should be fine.

I checked it, but it still doesn't work because `status-go` PR (with default `get.log`) is not added to this PR, if we use that branch I suppose it will work just fine. Currently I only confirmed that we need to return an empty string here as otherwise we get that weird structure.